microk8s-resources/wrappers/microk8s-ctr.wrapper

```diff
@@ -1,27 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -eu
 
-export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
-ARCH="$($SNAP/bin/uname -m)"
-export IN_SNAP_LD_LIBRARY_PATH="$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$ARCH-linux-gnu:$SNAP/usr/lib/$ARCH-linux-gnu"
-
 source $SNAP/actions/common/utils.sh
 
-SNAPSHOTTER=$(snapshotter)
+use_snap_env
 
 exit_if_no_permissions
 
+export CONTAINERD_SNAPSHOTTER=$(snapshotter)
+
 if ! [ -e $SNAP_DATA/args/ctr ]
 then
   echo "Arguments file $SNAP_DATA/args/ctr is missing."
   exit 1
+fi
+
+declare -a args="($(cat $SNAP_DATA/args/ctr))"
+if is_strict
+then
+  "${SNAP}/bin/ctr" "${args[@]}" "$@"
 else
-  declare -a args="($(cat $SNAP_DATA/args/ctr))"
-  if is_strict
-  then
-    LD_LIBRARY_PATH="$IN_SNAP_LD_LIBRARY_PATH" CONTAINERD_SNAPSHOTTER="$SNAPSHOTTER" "${SNAP}/bin/ctr" "${args[@]}" "$@"
-  else
-    sudo -E LD_LIBRARY_PATH="$IN_SNAP_LD_LIBRARY_PATH" CONTAINERD_SNAPSHOTTER="$SNAPSHOTTER" "${SNAP}/bin/ctr" "${args[@]}" "$@"
-  fi
+  sudo -E "${SNAP}/bin/ctr" "${args[@]}" "$@"
```

Comment:
Don't we drop PATH and LD_LIBRARY_PATH here?

Reply:
dito